### PR TITLE
SOAP mock init

### PR DESCRIPTION
**Pull request by zarkone**
Simple SOAP mock service integration
https://github.com/xapix-io/firebird-next/issues/145

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 var express = require("express"),
 	bodyParser = require("body-parser"),
+    soap = require("strong-soap").soap,
```
Comment (mreinsch) [inline app.js]: please check the indentation for this file. It uses tabs, so we want to match that style.

File: app.js [hunk 2]
```diff
@@ -107,3 +108,41 @@ var server = app.listen(port, function() {
...
+
+
+// SOAP
+
+var BLZService = {
+    BLZService: {
+        BLZServiceSOAP12port_http: {
+
+            getBank: function(args) {
+                console.log("getBank.");
```
Comment (mreinsch) [inline app.js]: needed?

File: app.js [hunk 3]
```diff
@@ -107,3 +108,41 @@ var server = app.listen(port, function() {
...
+            var jsonResponse = {"zElement": true};
+            return jsonResponse;
+          }
+        }
+      }
+}
+
+
+var xml = require('fs').readFileSync('./soap-data/BLZServiceSOAP12Only.xml', 'utf8');
+// var xml = require('fs').readFileSync('./soap-data/doc-literal.xml', 'utf8');
```
Comment (mreinsch) [inline app.js]: better to avoid commented out code. Can we load both? If not, how can we make this configurable?

File: soap-data/BLZService.xml [hunk 4]
```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0"?><wsdl:definitions xmlns:wsdl="http://schemas.xmlsoap.org/wsdl/" xmlns="http://schemas.xmlsoap.org/wsdl/" xmlns:wsaw="http://www.w3.org/2006/05/addressing/wsdl" xmlns:http="http://schemas.xmlsoap.org/wsdl/http/" xmlns:tns="http://thomas-bayer.com/blz/" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:mime="http://schemas.xmlsoap.org/wsdl/mime/" xmlns:soap="http://schemas.xmlsoap.org/wsdl/soap/" xmlns:soap12="http://schemas.xmlsoap.org/wsdl/soap12/" targetNamespace="http://thomas-bayer.com/blz/">
```
Comment (mreinsch) [inline soap-data/BLZService.xml]: let's drop any files not needed, I think it's confusing otherwise.

**Comment by zarkone** [comment]
@mreinsch fixed

**Pull request merged**